eslint-config-5minds-typescript/README.md

```diff
@@ -0,0 +1,73 @@
+# ESLint config for 5Minds TypeScript projects
+
+- install with
+  ```
+  npm install --save-dev eslint-config-5minds-typescript
+  ```
+- install the necessary plugins and parsers with
+  ```bash
+  npm install --save-dev \
+    @typescript-eslint/parser \
+    @typescript-eslint/eslint-plugin \
+    eslint-plugin-6river \
+    eslint-plugin-import \
+    eslint-plugin-no-null
```

Comment:
Ich bekomm nach dem Installieren folgendes:

![Bildschirmfoto 2019-05-02 um 12 08 32](https://user-images.githubusercontent.com/15343316/57070212-ffaf7200-6cd6-11e9-9b49-6b827ed05e3f.png)


Wenn die gebraucht werden, sollten die hier als Dependency mit aufgeführt werden. Oder es sollte zumindest in der Readme ein Hinweis stehen.

Reply:
Problem ist hier vermutlich, dass `eslint-config-5minds` noch auf `eslint-config-airbnb` aufbaut, anstatt auf `eslint-config-airbnb-base`. Ich ändere das eben

Reply:
Jo, das war das Problem. Ist jetzt gefixt. hab das Paket jetzt mal in Version `1.0.0-alpha.5` gepublished

Reply:
Sieht gut aus. Linting funktioniert auch wie erwartet. 👍